### PR TITLE
Start testing the debugger with Galata

### PR DESCRIPTION
**Pull request by jtpio**
## References

Follow-up to #10331 

## Code changes

Add basic skeleton to start testing the debugger with Galata.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by jtpio** [comment]
Adding `xeus-python` to the ui-test CI workflow also adds the debugger sidebar to the right area:

image

And it looks like this might make `galata` time out when resetting the UI with `moveAllTabsToLeft`:

https://github.com/jupyterlab/jupyterlab/pull/10359/checks?check_run_id=2775841320

image


https://github.com/jupyterlab/galata/blob/d717e6a95f738679eaec86f1e5d799458ff584c4/packages/galata/src/galata.ts#L1795-L1812

**Comment by jtpio** [comment]
Adding `xeus-python` to the workflow requires updating some of the reference screenshots, since there is now a new item in the launcher:

image

**Comment by jtpio** [comment]
This also means that folks will need to install `xeus-python` locally if they want to run the UI tests locally.

**Comment by jtpio** [comment]
Marking as ready for review, since this PR does the basic setup to start testing the debugger.

We can start adding more tests and proper visual regression testing with reference screenshots in follow-up PRs.

**Comment by jtpio** [comment]
> This also means that folks contributing to JupyterLab will need to install `xeus-python` locally if they want to run the UI tests locally.

Setting back as draft to check using the `ipykernel` pre-release which has support for debugging.

**Comment by jtpio** [comment]
Also the xeus-python launcher item has the Python version in its display name, which is likely to change over time and would require updating the reference screenshots when that happens.

File: scripts/ci_install.sh [hunk 1]
```diff
@@ -55,6 +55,12 @@ if [[ $GROUP == js-debugger ]]; then
     pip install xeus-python">=0.9.0,<0.10.0"
 fi
 
+# The UI tests use ipykernel with support for debugging, to not
+# extra launcher items and other UI elements to the reference screenshots
+if [[ $GROUP == ui-tests ]]; then
```
Comment (jtpio) [inline scripts/ci_install.sh:60]: It will then be possible to remove this block when the final release of `ipykernel` 6 is out

**Comment by jtpio** [comment]
Ref screenshots updated: 

image

**Review by blink1073 — APPROVED** [review]
Looks great, thank you!

**Comment by jtpio** [comment]
Rebasing one more time since https://github.com/jupyterlab/jupyterlab/pull/10376 has been merged.

**Review by blink1073 — APPROVED** [review]
Thanks!

**Pull request merged**